src/map/status.cpp

```diff
@@ -10889,6 +10890,11 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 				val4 = 1;
 			tick_time = val2 * 1000; // [GodLesZ] tick time
 			break;
+		case SC_GRADUAL_GRAVITY:
+			val2 = 10 * val1;
+			val4 = tick / 1000;
+			tick_time = status_get_sc_interval(type);
```

Comment:
```suggestion
			tick_time = status_get_sc_interval(type);
			val4 = tick - tick_time; // Remaining time
```
For interval based SCs val4 should be the remaining time.